docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -117,9 +117,52 @@ vaccination campaign.
 2. This model uses country-level data, and cannot be used to model local measles
 outbreaks due to lack of vaccination in small communities.
 
+Some of the **assumptions** made in this model are:
+
+1. There is no data available for population in recovered state in GBD. 
+So, we considered all the population who do not have measles as susceptible  This includes both susceptible and recovered population.
+To compensate this, the incidence rate among susceptible and recovered population is used for transition.
+
+2. There is no data avaialable for remission rate in GBD. So a constant remission rate is 
+calculated from duration assumption of 10 days [GBD-2017-YLD-Capstone-Appendix-1]_.
```

Comment:
Let's say we calculate a constant rate from the assumption of an *average* 10 day duration.